calicoctl/calico_ctl/status.py

```diff
@@ -252,3 +273,42 @@ def pprint_bird_protocols(version):
         print str(x) + "\n"
     else:
         print results + "\n"
+
+
+def _gobgp(cmd):
+    if getattr(sys, 'frozen', False):
+        gobgp = sys._MEIPASS + "/gobgp"
+    else:
+        gobgp = "gobgp"
+    results = subprocess32.check_output([gobgp, "-j"] + cmd.split(" "))
+    return json.loads(results)
+
+
+def pprint_gobgp_protocols(version):
+    x = PrettyTable(["Peer address", "Peer type", "State", "Since", "Info"])
+
+    now = int(time.mktime(datetime.datetime.now().timetuple()))
+    try:
+        neighbors = _gobgp("neighbor -t ipv%d" % version)
+    except subprocess32.CalledProcessError:
+        print "Couldn't connect to gobgp."
+        return
+
+    for neighbor in neighbors:
+        # calico-bgp-daemon currently only supports node-to-node mesh type
```

Comment:
Is it a limitation on GoBGP side? if so, can you add a warning message at startup that only node-to-node mesh is supported with gobgp, because someone can use gobgp and then set node mesh off manually with calicoctl and that wouldn't work